src/app/body/team-details/github-integration/github-integration.component.ts

```diff
@@ -79,6 +79,32 @@ export class GithubIntegrationComponent implements OnInit {
     this.enableAddToken = true
   }
 
+  unLinkGithub(){
+    this.repoLoc='github';
+    this.repoLink="";
+    this.addProjLink(this.repoLink);
+  }
+
+  addProjLink(projLink: string){
+    this.repoLink=projLink;
+    this.enableLoader=true;
+    const organizationDomain = this.backendService.getOrganizationDomain();
+    const callable = this.functions.httpsCallable('teams/addProjLink');
+    callable({OrganizationDomain: organizationDomain, TeamName: this.team.TeamName, ProjLink: this.repoLink, ProjLocation: 'github'}).subscribe({
+      next: (data) => {
```

Comment:
can ProjLocation be emptied instead of 'github'

Reply:
Yes when we proceed to integrate a repo again it becomes empty